### DISCUSSION TITLE
Add option to starr a post in list of posts

### DISCUSSION THREAD

**Discussion by pitscher** [hugo-sid/hugo-blog-awesome#222] [Feature requests]
Hey, I just started to use this theme and I really like it.

I would like to have the option to _starr_ (hope this is the correct word to depict this 😄 ) single posts to attract a visitors attention.
The author can starr a post by setting the value of a new variable `isStarred` of a posts front matter to true.
This will cause a little star to appear right next to the title of a starred post in the list of posts.

**By the way:** I already implemented this for my blog.
Checkout the attached picture.
@hugo-sid linking you as the main maintainer - let me know if you would be happy to see a PR for this so that it can be added to the next release. 🤓 

![Screenshot 2024-08-26 at 20 36 01](https://github.com/user-attachments/assets/823bb625-6708-4254-b760-59c52eea0899)

**Comment by hugo-sid** [comment]
> Hey, I just started to use this theme and I really like it.

I'm glad to hear that, @pitscher 🎉 

> @hugo-sid linking you as the main maintainer - let me know if you would be happy to see a PR for this so that it can be added to the next release. 🤓

Having an option to starr a post is a great idea. Please create a PR for this.

**Reply by pitscher** [reply]
Here you go: https://github.com/hugo-sid/hugo-blog-awesome/pull/223
I also made sure the star is actually centered - missed to do this for my implementation. (You might see it if you take a close look at the picture I shared 😅).
